src/main/java/org/sonarlint/intellij/util/ResourceLoader.java

```diff
@@ -21,14 +21,12 @@
 
 import com.google.common.base.Preconditions;
 import com.google.common.io.ByteStreams;
-
-import javax.annotation.concurrent.ThreadSafe;
-import javax.swing.Icon;
-import javax.swing.ImageIcon;
 import java.io.IOException;
 import java.io.InputStream;
 import java.util.Map;
 import java.util.concurrent.ConcurrentHashMap;
+import javax.annotation.concurrent.ThreadSafe;
+import javax.swing.*;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Explicitly import the specific classes needed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/coding_rules#rule_key=squid%3AS2208)